src/components/screen/__tests__/SignUp.test.tsx

```diff
@@ -1,97 +1,348 @@
+      beforeAll(() => {
+        props = createTestProps();
+        component = createTestElement(<SignUp {...props} />);
+        testingLib = render(component);
+      });
+
+      it('should show error message when email validation fails during signUp', async () => {
+        const emailInput = testingLib.getByTestId('input-email');
+        await wait(() => expect(emailInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(emailInput, 'email@email');
+        });
+
+        const passwordInput = testingLib.getByTestId('input-password');
+        await wait(() => expect(emailInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(passwordInput, 'Abc123##');
+        });
+
+        const confirmPasswordInput = testingLib.getByTestId('input-confirm-password');
+        await wait(() => expect(confirmPasswordInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(confirmPasswordInput, 'Abc123##');
+        });
+
+        const nameInput = testingLib.getByTestId('input-name');
+        await wait(() => expect(nameInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(nameInput, 'dooboo');
+        });
+
+        const btnSignUp = testingLib.getByTestId('btn-sign-up');
+        await wait(() => expect(btnSignUp).toBeTruthy());
+        act(() => {
+          fireEvent.press(btnSignUp);
+        });
+
+        const errorText = testingLib.getByTestId('error-email');
+        await wait(() => expect(errorText).toBeTruthy());
+      });
+
+      it('should show error message when password validation fails during signUp', async () => {
+        const emailInput = testingLib.getByTestId('input-email');
+        await wait(() => expect(emailInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(emailInput, 'email@email.com');
+        });
+
+        const passwordInput = testingLib.getByTestId('input-password');
+        await wait(() => expect(emailInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(passwordInput, 'Abc');
+        });
+
+        const confirmPasswordInput = testingLib.getByTestId('input-confirm-password');
+        await wait(() => expect(confirmPasswordInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(confirmPasswordInput, 'Abc');
+        });
+
+        const nameInput = testingLib.getByTestId('input-name');
+        await wait(() => expect(nameInput).toBeTruthy());
+        act(() => {
+          fireEvent.changeText(nameInput, 'dooboo');
+        });
+
+        const btnSignUp = testingLib.getByTestId('btn-sign-up');
+        await wait(() => expect(btnSignUp).toBeTruthy());
+        act(() => {
+          fireEvent.press(btnSignUp);
+        });
+
+        const errorText = testingLib.getByTestId('error-password');
+        await wait(() => expect(errorText).toBeTruthy());
+      });
+    });
+
+    it('should show error message when name validation fails during signUp', async () => {
```

Comment:
Same here as [above](https://github.com/dooboolab/hackatalk-mobile/pull/80/files#r361837623).

Reply:
should show error text when the name is not validated